ui/tests/integration/components/mfa-login-enforcement-form-test.js

```diff
@@ -141,7 +141,8 @@ module('Integration | Component | mfa-login-enforcement-form', function (hooks)
     await click('[data-test-mlef-save]');
     assert.true(this.didSave, 'onSave callback triggered');
     assert.strictEqual(this.model.name, 'bar', 'Name property set on model');
-    assert.strictEqual(this.model.mfa_methods.firstObject.id, '123456', 'Mfa method added to model');
+    const methods = await this.model.mfa_methods; //hasManyPromise
```

Comment:
If I call this.model.mfa_methods[0] I hit a promise with the methods firstObject on it (which won't be after the upgrade). This was a small change, but I'd keep an eye out for future test writing when accessing hasMany relationships like this.